api/app/controllers/api/v1/applicants_controller.rb

```diff
@@ -34,6 +39,17 @@ def update
                 render_error(applicant.errors)
             end
         end
+
+        # /applicants/delete or /applicants/:id/delete
+        def delete
+            params.require(:id)
+            applicant = Applicant.find(params[:id])
+            if session.destroy!
```

Comment:
Should this be `applicant.destroy!`?

Reply:
Yes. This is a mistake that carried over from PR #183. Once, I fix that PR, I'll rebase this branch off that PR, again.

Reply:
Fixed.